leetcode/src/997.c

```diff
@@ -0,0 +1,29 @@
+// Using hastable.

```

Comment:
Spelling mistake, I guess.
```suggestion
// Using hashtable.
```